src/stories/intercom.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import Intercom from "../components/intercom/index"
+
+export default {
+  title: "Intercom",
+  component: Intercom,
+  args: {
+    storyName: "Example",
+  },
+  argTypes: {
+    buttonName: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+}
+
+const Template = (args) => {
+  return (
+    <div className="mx-30 mb-40">
+      <div className="text-2xl mb-20">{args.storyName}</div>
+      <div className="w-12/12 md:w-3/12">
+        <Intercom
+          {...args}
+          label={("label", "Support")}
```

Comment:
this looks weird, can you check what this is supposed to do? :)

Reply:
To me it is just regular arguments so I rewrote it the "normal" way